src/parser/helper/Helper.ts

```diff
@@ -90,19 +90,28 @@ export function formatPrintPath(filePath: string, config: Configuration): string
 }
 
 /**
- * Checks whether the specified path represents a file with a file extension.
- * If so, this function returns a corresponding {@link ParseFile}-object that includes the file extension.
+ * Checks if the file extension of the specified path is a known file extension of a programming language.
+ * If so, this function returns a corresponding {@link ParseFile}-object that includes the language and file extension.
+ * Otherwise, the language is set to {@link Languages.Unknown}. The same applies if there is no file extension found,
+ * in which case the file extension is set to an empty string.
+ *
  * @param filePath Path that should be checked.
- * @return A ParseFile-object if there is a file extension, undefined if there is no file extension present.
+ * @return A ParseFile-object.
  */
-export function checkAndGetFileExtension(filePath: string): undefined | ParseFile {
+export function getLanguageFromFileExtension(filePath: string): ParseFile {
     const splitted = filePath.split(".");
     if (splitted.length >= 2) {
         const fileExtension = splitted[splitted.length - 1].toLowerCase();
+
         if (fileExtension.length > 0) {
-            return { fileExtension: fileExtension, filePath: filePath };
+            let language = fileExtensionToLanguage.get(fileExtension);
+            if (language === undefined) {
+                language = Languages.Unknown;
+            }
+            return { fileExtension: fileExtension, filePath: filePath, language: language };
         }
     }
+    return { fileExtension: "", filePath: filePath, language: Languages.Unknown };
```

Comment:
you should return { fileExtension: fileExtension, ......

Reply:
No, thats correct if there is no file extension